geode-core/src/main/java/org/apache/geode/internal/cache/BucketRegion.java

```diff
@@ -1261,7 +1333,14 @@ void basicUpdateEntryVersion(EntryEventImpl event) throws EntryNotFoundException
   }
 
   protected void distributeUpdateEntryVersionOperation(EntryEventImpl event) {
-    new UpdateEntryVersionOperation(event).distribute();
+    UpdateEntryVersionOperation op = new UpdateEntryVersionOperation(event);
+    long viewVersion = -1;
+    try {
+      viewVersion = op.startOperation();
```

Comment:
Out of curiosity, what happens if the startOperation() method returns a -1 (I think that may be possible based on how it's currently written or if an exception is thrown before token is assigned.  What is the expected way to handle endOperation with an invalid/-1 token?